test/check-machines-storage-pools

```diff
@@ -978,6 +978,56 @@ class TestMachinesStoragePools(VirtualMachinesCase):
         b.wait_in_text("#delete-tooltip", "Pool's volumes are used by VMs")
         b.mouse("#delete-pool-images-system a[aria-disabled=true]", "mouseleave")
 
+        # Detach the disk
+        m.execute("virsh detach-disk subVmTest1 vdb --config")
+        # Need to reload to update the disk usedby field
+        b.reload()
+        b.enter_page("/machines")
+
+        self.waitPoolRow("images")
+        self.togglePoolRow("images")
+        self.gotoVolumesTab("images")
+
+        b.wait_not_in_text("#pool-images-system-volume-forDeletion-usedby", "subVmTest1")
+
+        # Check for deleting a pool whose disk is used by a VM
+        b.click("#pool-images-system-action-kebab button")
+        b.click("#pool-images-system-action-kebab a:contains(Delete)")
+        b.set_checked("#storage-pool-delete-volumes", True)
+        b.wait_in_text(".pf-c-backdrop .pf-c-helper-text__item-text", "Pool's volumes are used by")
+        b.wait_visible(".pf-c-backdrop footer button[aria-disabled=true]:contains(Delete)")
+
+        # Check "Cancel" button
+        b.click(".pf-c-backdrop footer button:contains(Cancel)")
+        b.wait_not_present(".pf-c-backdrop")
+
+        # Check "X" button of the dialog
+        b.click("#pool-images-system-action-kebab button")
+        b.click("#pool-images-system-action-kebab a:contains(Delete)")
+        b.click(".pf-c-backdrop button[aria-label=Close]")
+        b.wait_not_present(".pf-c-backdrop")
+
+        # Check that the error shown after the title, not the footer
+        # Delete the VM, or the deletion button will be disabled
+        m.execute("virsh undefine subVmTest1")
+        # Check both command and UI that the VM is not shown
+        wait(lambda: "subVmTest1" not in m.execute("virsh list --all"))
+        b.click(".machines-listing-breadcrumb li a:contains(Virtual machines)")
+        b.wait_not_present("#vm-subVmTest1-system-name")
+        # Go back to the storage pool list
+        b.wait_in_text("#card-pf-storage-pools button .card-pf-title-link", "Storage pool")
+        b.click("#card-pf-storage-pools .pf-c-card__header button")
+        b.wait_visible("#storage-pools-listing")
+
+        m.execute(f'chattr +a {args["image"]}')
+        self.addCleanup(m.execute, f'chattr -a {args["image"]}')
+        b.click("#pool-images-system-action-kebab button")
+        b.click("#pool-images-system-action-kebab a:contains(Delete)")
+        b.set_checked("#storage-pool-delete-volumes", True)
+        b.click(".pf-c-backdrop footer button[aria-disabled=false]:contains(Delete)")
+        b.assert_pixels("div.pf-c-modal-box.pf-m-align-top.pf-m-sm", "storage-pool-deletion-error", skip_layouts=["rtl"])
```

Comment:
Can you remove the skip layouts? Any specific reason you put it or just wrong copy?

Reply:
Sure I can, the reason is that I am not sure where we will use the rtl layout. I will remove it.